chaincode/abstore/go/go.mod

```diff
@@ -1,12 +1,5 @@
-module github.com/hyperledger/fabric-samples/chaincode/abstore/go
+module github.com/Zzocker/fabric-samples/chaincode/abstore/go
 
-go 1.12
+go 1.14
```

Comment:
We need to keep this at 1.13 max for now for the chaincode to install successfully. Otherwise we run into the following error:

```
Error: chaincode install failed with status: 500 - failed to invoke backing implementation of 'InstallChaincode': could not build chaincode: docker build failed: docker image build failed: docker build failed: Error returned from build: 1 "go: inconsistent vendoring in /chaincode/input/src/github.com/hyperledger/fabric-samples/chaincode/abstore/go:
	github.com/hyperledger/fabric-contract-api-go@v1.0.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt

run 'go mod vendor' to sync, or use -mod=mod or -mod=readonly to ignore the vendor directory
```
This is because we currently don't package vendor/modules.txt with the chaincode. This is the same issue as described here: https://github.com/golang/go/issues/37948

So until we fix our chaincode packaging we need to avoid the consistency check by sticking to the go 1.13 spec in go.mod.

Reply:
thanks for linking the go package error.
resolved the issue